offices/_single.html

```diff
@@ -13,7 +13,7 @@
 
     {% from "post-macros.html" import activity_snippets as activity_snippets with context %}
     {% import "contact-macro.html" as contact_macro %}
-    {% set activities_feed = activity_snippets( office.tags if office.tags ) %}
+    {% set activities_feed = activity_snippets( office.tags if office.tags, include_date_flag=true) %}
```

Comment:
Can you also include this change on `sub-page/_single.html`? We won't be able to test it until Content tags all those pages, but that should also use the same approach as the office-page.